### DISCUSSION TITLE
Grid resolution and amount of particles

### DISCUSSION THREAD

**Discussion by bjmvandongen** [firemodels/fds#13326] [Q&A]
Hi all,

I have questions about two different cases. The first case I am modelling is one of the Catchpole experiments: EXMC8A. That is the experiment where a bed of excelsior is set on fire on a flat surface in a wind tunnel.  The fuel bed is 8 meters in the x-direction, , 1 meter in the y-direction and 0.152 m high.  I modelled the same experiment three times and tested three different grid sizes with a factor 8 difference between each simulation. The coarse grid has cell sizes of 0.1 x 0.1 x 0.1, the medium grid sizes of 0.05 x 0.05 x 0.05 and the fine grid 0.025 x 0.025 x 0.025 meters. The simulation mode that I used was LES. 
I do not understand some of my results, so I'm hoping someone can exlain it to me.

The first question is about the grid resolution. 
![Figure 2024-08-15 114737](https://github.com/user-attachments/assets/eb862ff8-ab88-4514-b029-b33ac65b8068)

As you can see in the graph above, the fire front in time is almost the same when using a coarse or medium grid. The fire propagation with medium grid is only a little bit faster after 200 seconds. What I do not understand is that the simulation with fine grid differs so much compared to the other two. **I was thinking that the fine grid should even be lying a little bit above the medium line, since that appeared to be the trend for the coarse and medium ones. How can this be?** **A finer grid should in principle give a more accurate solution right? Apart from the grid cell size, the one other thing that changed is the amount of particles, since I specified 1 particle per cell. Maybe the amount of particles could be the reason? If yes, why?**

The second question I have about this experiment is about the amount of particles when fixing the grid cell size. I took the fine grid and tested it with 89600 particles (the amount of particles you would have when specifying 1 particle per cell on a fine grid) and I tested it with 12800 particles (the amount of particles you would have when specifying 1 particle per cell on a medium grid). The results are shown below.

![particletestcatchpole](https://github.com/user-attachments/assets/deefd302-fc5a-41e5-bcc1-db6291635fce)

The first thing about this graph that I don't understand is that the simulation is so sensitive to the amount of particles that are used. **Is it a problem that with using a fine grid and only having 12800 particles, that not every cell has a particle?**
The second thing that I don't understand is that the green and red line differ. In both simulations a fine grid is used and the same amount of particles. The only difference between these simulations is the placement of the particles. For the green line, the particles are placed in the center of each cell and for the red line, the particles are placed randomly in the volume where the fuel should be. **Since particles are treated as volume effects, the placement should not matter right?**

Then, the second case is about the CSIRO grassland fires. I am simulation case F19. I used SVLES as simulation mode and the grid cell size was 0.5 x 0.5 x 0.5 meters. 
I tested the case on a fine grid with different amount of particles. The results are given below. 

![SVLES_fine_particletest](https://github.com/user-attachments/assets/0f5b4245-7a5e-46cb-aaf9-6bcf715ce93b)

As you can see, with two particles per cubic meter, the line follows the experimental line closely while for instance with 7 particles per cubic meter, the line deviates at 100 seconds. I looked at the simulation results in smokesview and then I saw that the fire front changed in shape. **With 7 particles/m^3, the fire propagates in the centerline the fastest, while with 2 p/m^3 the center fire slows down compared to the sides. Does anybody know why this happens?**

The fds input files for the catchpole and CSIRO grassfire experiments are attached below:
[EXMC8Afine.txt](https://github.com/user-attachments/files/16626684/EXMC8Afine.txt)
[CSIROfine2particles.txt](https://github.com/user-attachments/files/16626678/CSIROfine2particles.txt)

Then finally, a more general question about lagrangian particles. **What happens mathematically when placing more than one particle in a cell? Is the overall mass increased? Are some values higher?** I already know that the contribution of the radiation of the particles to the gas becomes higher because it is summed over all particles in the cell. I also think that the F_b term in the momentum equation becomes higher because it is summed over all particles in the cell. The  ˙m′′′b,α term in the mass trasnport equation is also higher then right? because it is also summed over all particles in the cell.
**I read in a thesis of someone that it doesn't really matter how many particles are placed in the cell. But the terms I mentioned above are affected right?**

**Comment by rmcdermo** [comment]
Welcome to research in wildland fire modeling.

These issues are why some of us (at least I) have been arguing for improvements to the so-called "physics-based" formulation.  The convergence properties of these methods are elusive.

> How can this be? A finer grid should in principle give a more accurate solution right?

For a simple PDE solved with a finite volume or finite difference method, this is of course true.  But that is not our situation here.  We have a very complicated hybrid multi-phase scheme where we make the disperse phase approximation (condensed phase takes up zero volume) and we add a complicated radiation solver on top of this.  The interactions and sensitivities are hard to pin down (to put it mildly).

> Apart from the grid cell size, the one other thing that changed is the amount of particles, since I specified 1 particle per cell. Maybe the amount of particles could be the reason? If yes, why?

Particles are processed serially and this has always led to an order of operations problem in the particle methods (originally designed to handle sprinkler droplets).  Again, particles also interact with the radiation field and this has its own time quirks in terms of time accuracy.

> Since particles are treated as volume effects, the placement should not matter right?

The position of particles affects the local velocity each particle sees and we do know that drag and heat transfer can be sensitive to these minor changes.

> With 7 particles/m^3, the fire propagates in the centerline the fastest, while with 2 p/m^3 the center fire slows down compared to the sides. Does anybody know why this happens?

Nominally, we would not expect 7 vs 2 particles to matter that much, but this speaks to the sensitivity of the schemes.  Again, convergence properties are not well understood with the full hybrid, multiphase methods.

> I read in a thesis of someone that it doesn't really matter how many particles are placed in the cell. But the terms I mentioned above are affected right?

If you put 1 or 2 particles in the cell and specify the BULK_DENSITY, then each particle is should weighted such at the total mass and energy are the same in each case.  So, mass and energy are conserved at the cell level, but how the fluxes behave may differ due to differences in position and order of processing of the particles.  These just are not completely solved problems at this stage.

**Reply by bjmvandongen** [reply]
Thank you very much for your response. A lot is more clear for me now. 
I do have a followup question. In this statement:
So, mass and energy are conserved at the cell level, but how the fluxes behave may differ due to differences in position and order of processing of the particles.

which fluxes do you mean? 
If i understand it correctly, the mass and energy at cell level are conserved such that it does not matter how many particles are in the cell. But since the particles are processed serially and are placed at different positions, the mass flux (˙m′′′b,α term), radiation and momentum are different at different positions in the cell?
